test/smoke/src/areas/positron/new-project-wizard/new-project.test.ts

```diff
@@ -4,74 +4,155 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { expect } from '@playwright/test';
-import { Application, Logger } from '../../../../../automation';
+import { Application, Logger, PositronPythonFixtures } from '../../../../../automation';
 import { installAllHandlers } from '../../../utils';
 
 /*
  * New Project Wizard test cases
  */
 export function setup(logger: Logger) {
 	describe('New Project Wizard', () => {
-		// Shared before/after handling
-		installAllHandlers(logger);
-
 		describe('Python - New Project Wizard', () => {
+			// Shared before/after handling
+			installAllHandlers(logger);
+
 			before(async function () {
 
 			});
 
 			it('Python Project Defaults [C627912]', async function () {
 				const app = this.app as Application;
-				await app.workbench.positronNewProjectWizard.startNewProject();
-				await app.workbench.positronNewProjectWizard.newPythonProjectButton.click();
-				await app.workbench.positronNewProjectWizard.projectWizardNextButton.click();
-				await app.workbench.positronNewProjectWizard.projectWizardNextButton.click();
-				await app.workbench.positronNewProjectWizard.projectWizardDisabledCreateButton.isNotVisible(500); // May need to pass in a retry count > default of 200
-				await app.workbench.positronNewProjectWizard.projectWizardNextButton.click();
-				await app.workbench.positronNewProjectWizard.projectWizardCurrentWindowButton.click();
+				const pw = app.workbench.positronNewProjectWizard;
+				await pw.startNewProject();
+				await pw.projectTypeStep.pythonProjectButton.click();
+				await pw.nextButton.click();
+				await pw.nextButton.click();
+				await pw.disabledCreateButton.isNotVisible(500);
+				await pw.nextButton.click();
+				await pw.currentOrNewWindowSelectionModal.currentWindowButton.click();
 				await app.workbench.positronExplorer.explorerProjectTitle.waitForText('myPythonProject');
 			});
 
+			describe('Python Project with existing interpreter', () => {
+				it('With ipykernel already installed [C609619]', async function () {
+					const projSuffix = '_ipykernelInstalled';
+					const app = this.app as Application;
+					const pw = app.workbench.positronNewProjectWizard;
+					const pythonFixtures = new PositronPythonFixtures(app);
+					// Start the Python interpreter and ensure ipykernel is installed
+					const interpreterInfo = await pythonFixtures.startAndGetPythonInterpreter(true);
+					expect(interpreterInfo?.path).toBeDefined();
+					const interpreterPath = interpreterInfo?.path!;
```

Comment:
This ends up causing a failure for me on windows.. because `interpreterPath` is something like `~\pathtopython` but.. when the dropdown in the New Project pane has `C:\Users\JonV\pathtopython`. It's an issue with the `~` expansion in windows (or lack thereof)